tests/serve/tito_parity/launch_dynamo_disagg.sh

```
@@ -0,0 +1,69 @@
+if [[ $# -ne 4 ]]; then
+    echo "Usage: $0 MODEL ENGINE_CONFIG STARTUP_TIMEOUT READY_FILE" >&2
+    exit 2
+fi
+
+MODEL=$1
+ENGINE_CONFIG=$2
+STARTUP_TIMEOUT=$3
+READY_FILE=$4
+
+SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+source "$SCRIPT_DIR/../../../examples/common/launch_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../../examples/common/launch_utils.sh was not specified as input (see shellcheck -x).

(SC1091)